tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -809,12 +809,10 @@ describe("Service Form Modal", function() {
           });
         });
 
-        it('should disable the "Advanced Settings" section', function() {
-          cy.contains("Advanced Settings").parent().within(function() {
-            cy
-              .get("label.form-control-toggle.disabled")
-              .should("have.length", 2);
-          });
+        it('should disable the "Grant Runtime Privileges" checkbox', function() {
```

Comment:
We should actually change the test case description to `it disables the ...` following betterspecs.org but this shouldn't block the PR.